packages/react/src/settings/hooks/__tests__/useConfiguration.test.tsx

```diff
@@ -0,0 +1,128 @@
+import { cleanup, renderHook } from '@testing-library/react';
+import { fetchConfiguration } from '@farfetch/blackout-redux';
+import {
+  mockConfigurationCode,
+  mockConfigurationErrorState,
+  mockConfigurationLoadingState,
+  mockConfigurationsInitialState,
+  mockConfigurationsState,
+} from 'tests/__fixtures__/settings';
+import { useConfiguration } from '../..';
+import { withStore } from '../../../../tests/helpers';
+
+jest.mock('@farfetch/blackout-redux', () => ({
+  ...jest.requireActual('@farfetch/blackout-redux'),
+  fetchConfigurations: jest.fn(() => () => Promise.resolve()),
```

Comment:
This mock is needed here? I believe not because in not used on the hook file :) What do you think?

Reply:
yes, correct! I'v removed it :) Thanks